### ISSUE TITLE
`LICENSE` misssing from 2.15.1 jar

### ISSUE THREAD

**Issue by pjfanning** [FasterXML/jackson-core#1032]
Was removed by https://github.com/FasterXML/jackson-core/commit/c04e9243bda008ac570fb5da8def07163eae1749#diff-8eac27e2b7403d363ecdb0a19ea08fafe1aa78484e74fb74664d50f202bc2e94

I think it would be better to have the LICENSE in the jar. The one that was removed by the commit was probably fine.

**Comment by cowtowncoder** [comment]
@pjfanning There is root-level `LICENSE` that "build-helper-maven-plugin" used to include. But maybe later changes wrt license processing broke this somehow? Ideally we would not have 2 LICENSES: main level one is needed for Github UI so that seemed like better choice than something under ``src/main/resources/META-INF/LICENSE`.

Either way, yes, we do want LICENSE included and its dropping was not intentional

**Comment by cowtowncoder** [comment]
Looks like 2.15.0 jar does have `META-INF/LICENSE` but 2.15.1 not. I think something with added post-processing (to produce `META-INF/FastDoubleParser-LICENSE` and `META-INF/bigint-LICENSE`) broke inclusion of "main" LICENSE. :-(

**Comment by pjfanning** [comment]
I tried a few maven plugins but still struggling to get the LICENSE included. Is it ok if I just copy it to src/resources/META-INF again? This change is still going to be painful because FastDoubleParser has clashing files that we need to keep (but rename).

I already had to rename the NOTICE as src/resources/META-INF/jackson-core-NOTICE and rename it as part of the maven shade task execution. The plan would to do something similar for the LICENSE file.
